extension/pgtelemetry--1.3.sql

```diff
@@ -0,0 +1,512 @@
+-- complain if script is sourced in psql, rather than via CREATE EXTENSION
+--\echo Use "CREATE EXTENSION pgtelemetry" to load this file. \quit
+
+-- filter non-backend connections from pg_stat_activity
+create view client_stat_activity as select * from pg_stat_activity where backend_type = 'client backend';
+
+-- disk space
+CREATE VIEW relation_total_size AS
+select c.oid, c.oid::regclass as relation, pg_total_relation_size(c.oid) as inclusive_bytes,
+       pg_size_pretty(pg_total_relation_size(c.oid)) as inclusive_size,
+       pg_relation_size(c.oid) as exclusive_bytes,
+       pg_size_pretty(pg_relation_size(c.oid)) as exclusive_size
```

Comment:
I'd avoid calling `pg_*_relation_size` functions twice. it's slow

Reply:
thats not part of my patch though;

Reply:
I don't get it: it seems we only had 1.2 before you patch, why not adding it to 1.3 and releasing 1.3?

Reply:
because 1.3 was never merged for no apparent reason. I noticed it a bit too late unfortunately